.github/CONTRIBUTING.md

```diff
@@ -20,6 +20,15 @@ Help us keep CakePHP open and inclusive. Please read and follow our [Code of Con
   * Clearly describe the issue including steps to reproduce when it is a bug.
   * Make sure you fill in the earliest version that you know has the issue.
 * Fork the repository on GitHub.
+* Follow the steps in [README.md](https://github.com/MESH-Research/CCR/blob/master/README.md) to setup a development environment with [Lando](https://lando.dev).
+
+## Branches and Release Flow
+
+* The `development` branch is our bleeding edge.
+  * All commits on this branch will follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (CC)
+  * Pull requests into this branch will be squashed into a CC or rebased if all commits are CC and the detail is needed for CHANGELOG genereation or release versioning.
```

Comment:
The way this is phrased is a bit confusing. Will a rebase occur if detail is needed for the CHANGELOG or release versioning? Maybe break it up into two sentences to make this more clear?

Reply:
I fixed the phrasing in an additional commit.  Let me know if it still reads awkwardly.

Reply:
Yeah, it's still unclear, particularly '... the detail is needed for CHANGELOG generation or release versioning'.

To avoid additional ambiguity, I would also suggest reversing the clauses of both of those sentences so that 'the PR may be rebased' and 'the PR should be squashed and merged' are in the first parts of those sentences, respectively.

Here's what I suggest changing these to, if I understand their message correctly:

> * The PR may be rebased if all its commits follow Conventional Commits guidelines and there is a need of additional detail for CHANGELOG generation or release versioning.
> * Otherwise, the PR should be squashed and merged if its commits can be squashed into a single commit.

What do you think?

Reply:
Implemented.